### PR TITLE
Fix doc build on main

### PR DESCRIPTION
**Pull request by hoechenberger**
We broke Circle builds by merging #9099. This PR attempts to fix the problem.

### CODE REVIEW

**Comment by hoechenberger** [comment]
@drammock Any idea why Circle won't fire up here? Or do I just need to be more patient?

File: examples/preprocessing/plot_run_ica.py [hunk 1]
```diff
@@ -46,7 +46,8 @@
 # Fit ICA model using the FastICA algorithm, detect and plot components
 # explaining ECG artifacts.
 
-ica = ICA(n_components=0.95, method='fastica').fit(epochs)
+ica = ICA(n_components=0.95, method='fastica', max_iter='auto')
```
Comment (larsoner) [inline examples/preprocessing/plot_run_ica.py:49]:
```suggestion
ica = ICA(n_components=0.95, method='fastica', max_iter='auto',
          random_state=0)
```

**Comment by larsoner** [comment]
I really just pushed because circle wasn't building your commit with a weird message about a policy violation

https://app.circleci.com/pipelines/github/mne-tools/mne-python/7200/workflows/94e2cb9f-495a-4d34-b838-f72a8a8a1a65/jobs/26235

I want to see if my commit has the same issue, other builds don't seem to have it. @hoechenberger you might want to email the address they mention in that build to ask what's going on

**Comment by larsoner** [comment]
Indeed my commit is already restoring caches so it must be something about your account/commits that they errantly flag it

**Comment by larsoner** [comment]
Thanks @hoechenberger !

**Comment by hoechenberger** [comment]
Thanks @larsoner , I will get in touch with Circle folks to get this sorted out

**Pull request merged**